apps/website/src/pages/docs/expandableblock.mdx

```diff
@@ -9,67 +9,83 @@ group: core
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-expandableblock' />
-
 <LiveExample src='ExpandableBlock.main.tsx'>
   <AllExamples.ExpandableBlockMainExample client:load />
 </LiveExample>
 
-Expandable blocks, called expandable rows when inside a table, are used to progressively disclose information. Read this article to learn more about progressive disclosure. Expandable blocks and rows should not be used with flattened hierarchy or information panels. This type of navigation allows the user to see content inside a parent row or block.
+The expandable block has a label reflecting its content. On click, it expands to reveal more content.
+
+## Variants
+
+### With caption
+
+Users can optionally display descriptor text underneath the title.
+
+<LiveExample src='ExpandableBlock.withcaption.tsx'>
+  <AllExamples.ExpandableBlockWithCaptionExample client:load />
+</LiveExample>
 
-## Appearance
+### Accordion
 
-### Expandable rows
+Expandable blocks can also stack on top of each other in an accordion to make large sets of information easier to read.
 
-Expandable rows resemble normal table rows that can be expanded to reveal nested data. An expandable row could optionally be selectable. Because expandable rows are part of a table, the rows must sort based on the table column sort applied. This means expandable rows can not re-arranged, the user must interact with the table header to re-arrange.
+<LiveExample src='ExpandableBlock.accordion.tsx'>
+  <AllExamples.ExpandableBlockAccordionExample client:load />
+</LiveExample>
 
-Expandable rows should not be confused with a tree hierarchy. Unlike trees, they follow a single branch pattern. If you have trouble telling the two apart, the Expandable block VS Hierarchy tree section explains what makes them different.
+### With status
 
-There can be several child elements nested in an expandable row. These indented elements all coexist on the same level without further indentation. If you require more levels, a tree hierarchy is a better pattern for this interaction.
+An expandable block could display status, an icon, or other helpful information at a glance right aligned within the block header.
 
-The child elements don’t reflect the column titles. They are expected to retain the same properties as the parent row (in the example below, the owner). It’s unnecessary to repeat the properties for the child items because of this, so do not include them in the expanded row area.
+<LiveExample src='ExpandableBlock.status.tsx'>
+  <AllExamples.ExpandableBlockStatusExample client:load />
+</LiveExample>
```

Comment:
This one also has the problem with not being scrollable: 
![image](https://user-images.githubusercontent.com/37936169/230216288-cf0aeb5e-9ea5-4b9f-a9af-4c0195f0fefe.png)

Reply:
Fixed